app/uk/gov/nationalarchives/omega/editorial/services/EditSetPagination.scala

```diff
@@ -57,10 +57,10 @@ class EditSetPagination(
         (None, None)
 
       case 1 =>
-        (None, nextLink(numberOfPages))
+        (None, nextLink(currentPage + 1))
```

Comment:
Was this fix related to a different bug? I didn't see any relevant changes to the tests.